roles/installer/ospd/images/build/tasks/main.yml

```diff
@@ -1,2 +1,29 @@
 ---
-# TODO: Add images build code here
+- fail: msg="Check is image build path is defined"
```

Comment:
please remove this line, it doesn't make sense, this will fail either way in the next task if image build path doesn't exist